examples/jsm/webxr/XRPlaneUtils.js

```diff
@@ -0,0 +1,145 @@
+import {
+	Matrix4,
+	Vector3
+} from 'three';
+
+const UncategorizedPlaneType = - 1;
+const WallPlaneType = 2;
+const FloorPlaneType = 3;
+const CeilingPlaneType = 4;
+const PlatformPlaneType = 5;
+
+class XRPlaneUtils {
+
+	constructor( renderer ) {
+
+		this.renderer = renderer;
+    }
+
+    getPlaneType( plane ) {
+
+        const planes = this.renderer.xr.getPlanes()
+
+        const planeCenterPoint = new Map();
+        const planeNormal = new Map();
+        const planeHorizontal = new Map();
+
+        const xrReferenceSpace = this.renderer.xr.getReferenceSpace();
+        const xrFrame = this.renderer.xr.getFrame();
+
+        planes.forEach( plane => {
```

Comment:
Why do we need to parse all the planes in the session *every time* we wants to know the type of just one plane?

Reply:
Never mind.